src/modules/addtask.mjs

```diff
@@ -0,0 +1,45 @@
+import remove from './remove.mjs';
+import editTask from './editDesc.mjs';
+import toLS from './saveLocalStorage.mjs';
+
+const addDiv = document.getElementById('add');
+const input = document.getElementById('addTask');
+
+// CreateTask
+const newTask = (name, index) => {
+  const task = {
+
+    name,
+    completed: false,
+    index,
+  };
+  return task;
+};
+
+// Add to the list
+const addToLi = (name, index, trashImg, taskList, todoList) => {
+  const element = document.createElement('li');
+  element.classList = 'task';
+
+  element.innerHTML = `<input type="checkbox" name="${index}"><label for="${index}">${name}</label><div class="remove"><img src=${trashImg} alt="remove"/>`;
+  todoList.appendChild(element);
+  remove(element, index, taskList, todoList);
+  editTask(element, taskList);
+  checkBox(element, index, taskList);
+};
+
+// Add task to the list and display
+const listen = (taskList, trashImg, todoList) => {
+  addDiv.addEventListener('submit', (e) => {
+    e.preventDefault();
+    if (input.value) {
+      addToLi(input.value, taskList.length, trashImg, taskList, todoList);
+      taskList.push(newTask(input.value, taskList.length));
+
+      input.value = '';
+      toLS(taskList);
+    }
+  });
+};
```

Comment:
- [ ] Following the project requirements, all changes to the To-Do List (adding tasks, deleting tasks, editing task descriptions) should be saved in local storage. I added 3 tasks but they were not added to the local storage. Kindly work on that. I would recommend using dev tools to check the status of the tasks.

<img width="958" alt="image" src="https://user-images.githubusercontent.com/77190203/161095553-d0265c82-c4c7-4805-9863-c926b7ce674e.png">